docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -133,6 +133,8 @@ not yet been retrieved into the local copy. If that happens, {es} will eagerly
 retrieve the data needed to complete the search in parallel with the ongoing
 recovery. On-disk data is preserved across restarts, such that the node does
 not need to re-download data that is already stored on the node after a restart.
+Note that the underlying snapshot still needs to be always available and reliable
+for a fully-mounted index (see <<searchable-snapshots-reliability>>).
```

Comment:
I understand why you'd like this to be here, but I would still rather remove it. There are many warnings in the manual, but if we go down the path of adding a link to them every time a user didn't read all the relevant info then the docs become rather hostile (and we will still continue to encounter users that don't read all the relevant info despite the extra links).

There is already a link to this information in the sidebar on the right:

<img width="1155" alt="image" src="https://user-images.githubusercontent.com/5058284/213146762-c3de889c-6c42-48a5-b8c3-ff7a7bed0367.png">

Maybe there's a different name for this subsection that would help draw attention to it?

Reply:
OK, I can remove the forward reference.

> Maybe there's a different name for this subsection that would help draw attention to it?

You mean instead of `Reliability of searchable snapshots` to rename it? Maybe something like `Underlying repository snapshots must be available and reliable` ?